### PR TITLE
Align the Read More button in the Actions List block

### PR DESCRIPTION
**Pull request by mardelnet**
Ref: https://greenpeace-gpi.slack.com/archives/C0160MX64AG/p1726049842778739

<hr>

**DESCRIPTION:**
- This PR limits the number of lines of the Actions List's titles to 2.
- This PR aligns the position of the Read More button of the Actions List block according to this mockup:
Frame 574

<hr>

**COMPARISON:**

Main branch:
Greenpeace

This branch:
Greenpeace(1)

### CODE REVIEW

File: assets/src/scss/blocks/ActionsList/ActionsListStyle.scss
Comment (mleray) [inline assets/src/scss/blocks/ActionsList/ActionsListStyle.scss:85]: For RTL screens it should be aligned to the left instead

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [pluto](https://www-dev.greenpeace.org/test-pluto) | [admin](https://www-dev.greenpeace.org/test-pluto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-pluto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-pluto) | [composer-local.json](https://github.com/greenpeace/planet4-test-pluto/blob/main/composer-local.json)

:watch: 2024.09.18 08:55:33

File: assets/src/scss/blocks/ActionsList/ActionsListStyle.scss
Comment (mardelnet) [reply assets/src/scss/blocks/ActionsList/ActionsListStyle.scss:85]: Nice catch!
I think this is the first time in my life I have to consider an RTL screen.

Comment (mardelnet) [reply assets/src/scss/blocks/ActionsList/ActionsListStyle.scss:85]: Testing page: https://www-dev.greenpeace.org/test-pluto/actions-list-test-page/

Comment (mleray) [reply assets/src/scss/blocks/ActionsList/ActionsListStyle.scss:85]: We have a few NROs that use RTL languages, so we need to think about it 🙂

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Comment by mardelnet** [comment]
> Looks good to me 👍

@mleray I've dismissed your approval since there's a new requirement:
"Limit the number of lines of the Actions List's titles to 2"

**Comment by mleray** [comment]
Right now it looks a bit off if there are 2 lines for the title and 3 lines for the description - see this [page](https://www-dev.greenpeace.org/test-pluto/actions-list/) for example.

**Comment by mardelnet** [comment]
> Right now it looks off in bigger screens if there are 2 lines for the title and 3 lines for the description - see this [page](https://www-dev.greenpeace.org/test-pluto/actions-list/) for example. I guess we just need to update the card's height accordingly, but that should be checked with the design team!

Good catch, @mleray 
Please, check the updates: https://www-dev.greenpeace.org/test-pluto/actions-list/

**Review by mleray — APPROVED** [review]
Looks good to me 👏

**Comment by mardelnet** [comment]
> Looks good to me 👏

Sorry, @mleray 
I have to dismiss your approval for a second time since new design changes were asked. Instead of increasing the box height for big screens, we are reducing the separation between titles and descriptions for all screens, consistently.
Don't worry, UAT is passed now :)

File: assets/src/scss/blocks/ActionsList/ActionsListStyle.scss
Comment (mleray) [inline assets/src/scss/blocks/ActionsList/ActionsListStyle.scss:105]: We could use `$sp-1` here

Comment (mardelnet) [reply assets/src/scss/blocks/ActionsList/ActionsListStyle.scss]: Done!

**Pull request merged**
